tests/test_cli.py

```diff
@@ -327,3 +331,77 @@ def test_multiple_folders(script_runner):
     assert ret.success, "process should return success"
     assert ret.stderr == '', "stderr should be empty"
     assert "full bbox" in ret.stdout, "joined bboxes of all files inside folder are printed to console"
+
+
+def test_zenodo_valid_link_repository(script_runner):
+    ret = script_runner.run('geoextent',
+                            '-b', '-t', 'https://zenodo.org/record/820562')
+    assert ret.success, "process should return success"
+    assert 'has no identifiable time extent' in ret.stderr
+    result = ret.stdout
+    bboxList = parse_coordinates(result)
+    assert bboxList == pytest.approx([96.21146, 25.55834, 96.35495, 25.63293], abs=tolerance)
+    assert "4326" in result
+
+
+def test_zenodo_valid_doi_repository(script_runner):
+    ret = script_runner.run('geoextent',
+                            '-b', '-t', 'https://doi.org/10.5281/zenodo.820562')
+    assert ret.success, "process should return success"
+    assert 'has no identifiable time extent' in ret.stderr
+    result = ret.stdout
+    bboxList = parse_coordinates(result)
+    assert bboxList == pytest.approx([96.21146, 25.55834, 96.35495, 25.63293], abs=tolerance)
+    assert "4326" in result
+
+
+def test_zenodo_valid_link_repository_with_no_geoextent(script_runner):
+    ret = script_runner.run('geoextent', '-b', '-t', 'https://zenodo.org/record/1810558')
+    result = ret.stdout
+    assert "bbox" not in result, "This repository contains a PDF file, it should not return a bbox"
+    assert "tbox" not in result, "This repository contains a PDF file, it should not return a tbox"
+
+
+def test_zenodo_invalid_link_repository(script_runner):
+    ret = script_runner.run('geoextent',
+                            '-b', '-t', 'https://zenado.org/record/820562')
+    assert not ret.success, 'Typo in URL'
+    assert "is not a valid" in ret.stderr, 'Typo in URL'
+
+
+def test_zenodo_valid_but_removed_repository(script_runner):
+    ret = script_runner.run('geoextent', '-b', '-t', 'https://zenodo.org/record/1')
+    assert not ret.success
+    assert "does not exist" in ret.stderr
+
+
+def test_zenodo_valid_but_no_extraction_options(script_runner):
+    ret = script_runner.run('geoextent', 'https://zenodo.org/record/1')
+    assert not ret.success, 'No extractions options, geoextent should fail'
+    assert "Require at least one of extraction options, but bbox is False and tbox is False" in ret.stderr
+
+
+def test_zenodo_valid_but_not_open_access(script_runner):
+    ret = script_runner.run('geoextent', '-b', '-t', 'https://zenodo.org/record/51746')
+    assert not ret.success, 'The repository exists but it is not accessible. Geoextent should fail'
+    assert "This record does not have Open Access files. Verify the Access rights of the record" in ret.stderr
+
+
+def test_export(script_runner):
```

Comment:
- Should we validate/test the file name (that it ends in "gpkg"?).
- What else can go wrong here?
- Please add tests for relative path, e.g., `path/to/output/file.gpkg` (error if target directory does not exist)
- Please add test for absolute path, e.g., `/tmp/out.gpkg` (you should get the full path from the `tmp` variable)
- When the file exists, we will just overwrite it, right? Please add a test for it (i.e. no problem when the file exists, just create an empty file of that name)

Reply:
>  Should we validate/test the file name (that it ends in "gpkg"?)
Geoextent raise a warning "The filename extension should be 'gpkg' instead of 'abc' to conform to the GPKG specification". Nevertheless the file is created. This is the standard behavior of GDAL.

Reply:
OK - good behaviour and warning message, please add tests :-)